devel/nodejs12/Portfile

```diff
@@ -86,8 +86,7 @@ universal_variant       no
 configure.ccache        no
 
 test.run                yes
-
-use_parallel_build      no
+test.cmd                ${build.cmd} -j${build.jobs}
```

Comment:
Is this actually the right command for running a test?

Reply:
https://github.com/nodejs/node/blob/master/BUILDING.md#running-tests recommends `make -j4 test`. From my understanding of Portfile syntax default `build.cmd` for `test.cmd` here is `make test`, so this should result in `make test -j{NUM_OF_CPU_CORES}`.

I tested it on my system and in `ps aux` I saw `make test -j8`.

Reply:
Ah. Okay. My confusion then.